example.yaml

```diff
@@ -71,6 +71,19 @@ data:
   # base64 encoded: my super cool \n multiline \ secret
   secret.world: bXkgc3VwZXIgY29vbAptdWx0aWxpbmUKc2VjcmV0
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: output-of-command
+  labels:
+    findme: "yup"
+data:
+  rand.sh: |
+    #!/bin/sh
+    dd if=/dev/random bs=4 count=1 | hexdump -v -e '/1 "%02X"'
+  random.txt.command: '/tmp/rand.sh'
```

Comment:
I think this example is only reliable if the key-values of a configmap are actually sorted when accessed through the python client. Otherwise occasionally the command is executed before the shell script is stored. Could you double check this?